### PR TITLE
feat(cicd): upd github workflows

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. metriport/metriport-internal#1251

### Description

- Updated the Github workflows to include the new packages

### Release Plan

- Nothing special

### CODE REVIEW

File: .github/workflows/deploy-production.yml [hunk 1]
```diff
@@ -39,10 +40,12 @@ jobs:
           base: "master"
           filters: |
             api:
+              - "packages/shared/
               - "packages/api/**"
               - "packages/api-sdk/**"
               - "packages/commonwell-sdk/**"
               - "packages/core/**"
+              - "packages/ihe-gateway-sdk/**"
```
Comment (leite08) [inline .github/workflows/deploy-production.yml:48]: Changes on this package don't need to result in a new deploy of the API, right?

File: .github/workflows/deploy-production.yml [hunk 2]
```diff
@@ -39,10 +40,12 @@ jobs:
           base: "master"
           filters: |
             api:
+              - "packages/shared/
```
Comment (leite08) [inline .github/workflows/deploy-production.yml:43]: Changes on this package don't need to result in a new deploy of the API, right?

File: .github/workflows/deploy-staging.yml [hunk 3]
```diff
@@ -38,10 +39,12 @@ jobs:
         with:
           filters: |
             api:
+              - "packages/shared/
               - "packages/api/**"
               - "packages/api-sdk/**"
               - "packages/commonwell-sdk/**"
               - "packages/core/**"
+              - "packages/ihe-gateway-sdk/**"
```
Comment (leite08) [inline .github/workflows/deploy-staging.yml:47]: Ditto Changes on this package don't need to result in a new deploy of the API, right?

File: .github/workflows/deploy-production.yml (hunk 2 above)
Comment (leite08) [reply .github/workflows/deploy-production.yml]: they do

**Review by leite08 — APPROVED** [review]

**Comment by RamilGaripov** [comment]
Updated a slight error in this PR with this commit - https://github.com/metriport/metriport/commit/e932fe0c8ee3dd1eb3bb0e0f9aa33f313c551caf

**Pull request merged**